DigitalLearningSolutions.Web/Controllers/FrameworksController/FrameworksController.cs

```diff
@@ -2,13 +2,18 @@
 {
     using DigitalLearningSolutions.Data.ApiClients;
     using DigitalLearningSolutions.Data.DataServices;
-    using DigitalLearningSolutions.Data.Services;
-    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
     using Microsoft.AspNetCore.Authorization;
     using Microsoft.AspNetCore.Mvc;
-    using Microsoft.Extensions.Logging;
-
-    [Authorize(Policy = CustomPolicies.UserFrameworksAdminOnly)]
+    using Microsoft.Extensions.Logging;
+
+    [Authorize(Policy = CustomPolicies.UserFrameworksAdminOnly)]
+    [SetDlsSubApplication(nameof(DlsSubApplication.Frameworks))]
+    [SetSelectedTab(nameof(NavMenuTab.Dashboard))]
```

Comment:
This is needed for all controller get methods in Frameworks (there are several other files that extend the FrameworksController class in FrameworksController/ )